includes/admin/class-amp-template-customizer.php

```diff
@@ -330,8 +330,12 @@ public function add_customizer_scripts() {
 		$dependencies = $asset['dependencies'];
 		$version      = $asset['version'];
 
-		/** This action is documented in includes/class-amp-theme-support.php */
-		do_action( 'amp_register_polyfills' );
+		if ( ! Services::get( 'dependency_support' )->has_support() ) {
```

Comment:
Aside: This is essentially re-making the `Polyfills` service conditional which was removed in https://github.com/ampproject/amp-wp/pull/7421

Interestingly, also, there are some cases where this will never be reachable:

https://github.com/ampproject/amp-wp/blob/3c17ea98510c637c671d88810c11e057ac71c5b0/includes/admin/functions.php#L18-L54

Reply:
>Aside: This is essentially re-making the Polyfills service conditional which was removed in https://github.com/ampproject/amp-wp/pull/7421

But only for customizers right?

> Interestingly, also, there are some cases where this will never be reachable:

Actually, added it as a safeguard. Should we remove it? What if it surpassed the condition in any case?

Reply:
I'm unsure. Maybe what we should do is just change `amp_init_customizer` to just `return` early if `! Services::get( 'dependency_support' )->has_support()`. 

And then at the same time, we can _remove_ all of the `amp_register_polyfills` actions from this class. What this means is that the custom AMP Customizer integrations won't be available on WP<5.6, but that's effectively the earliest version we support anyway. 

Additionally, Customizer usage is declining.

Reply:
So this means replacing all of the contents of this `if` condition:

https://github.com/ampproject/amp-wp/blob/3c17ea98510c637c671d88810c11e057ac71c5b0/includes/admin/functions.php#L20-L51

With just `return`.

Reply:
I think you mean:

```php
if ( ! Services::get( 'dependency_support' )->has_support() ) {
		return;
}
```

We can also remove the `@codeCoverageIgnore` right?

Reply:
Yes, that's right.

Reply:
Or rather, you can make it:

```php
if ( ! Services::get( 'dependency_support' )->has_support() ) {
    return; // @codeCoverageIgnore
}
```